app/validators/blocks/list_collector_validator.py

```diff
@@ -6,27 +6,30 @@
 
 class ListCollectorValidator(BlockValidator, ValidateListCollectorQuestionsMixin):
     LIST_COLLECTOR_KEY_MISSING = "Missing key in ListCollector"
-    ADD_ANSWER_REFERENCE_NOT_IN_MAIN_BLOCK = (
-        "add_answer reference uses id not found in main block question"
-    )
+    ADD_ANSWER_FOR_LIST_ITEM = "AddAnswerForListItem"
+    REMOVE_ANSWER_FOR_LIST_ITEM = "RemoveAnswerForListItem"
+
     NO_RADIO_FOR_LIST_COLLECTOR = (
         "The list collector block does not contain a Radio answer type"
     )
+
     NO_RADIO_FOR_LIST_COLLECTOR_REMOVE = (
         "The list collector remove block does not contain a Radio answer type"
     )
-    NON_EXISTENT_LIST_COLLECTOR_ADD_ANSWER_VALUE = (
-        "The list collector block has an add_answer_value that is not "
-        "present in the answer values"
+
+    ADD_ANSWER_FOR_LIST_ITEM_ACTION_NOT_IN_MAIN_BLOCK = (
+        "AddAnswerForListItem action not found in main block"
     )
-    NON_EXISTENT_LIST_COLLECTOR_REMOVE_ANSWER_VALUE = "The list collector block has a remove_answer_value that is not present in the answer values"
-    REMOVE_ANSWER_REFERENCE_NOT_IN_REMOVE_BLOCK = (
-        "remove_answer reference uses id not found in remove_block"
+
+    REMOVE_ANSWER_FOR_LIST_ITEM_ACTION_NOT_IN_REMOVE_BLOCK = (
```

Comment:
I think it would be clearer to follow the error name/text used elsewhere in the class for these messages. e.g `NO_ADD_ANSWER_ACTION_FOR_LIST_COLLECTOR = "The list collector main block does not contain an AddAnswerForListItem action"`

Reply:
True, I actually just changed the one that was already there and didn't think much of it.